Gdk-4.0.gir

```diff
@@ -19385,59 +19821,6 @@ of text, such as when text is selected.</doc>
         </parameter>
       </parameters>
     </function>
-    <function name="pixbuf_get_from_surface" c:identifier="gdk_pixbuf_get_from_surface">
```

Comment:
That this function disappears is problematic @bilelmoussaoui

Reply:
that was fixed iirc